security/cert_manager_operator/cert-manager-operator-release-notes.adoc

```diff
@@ -10,11 +10,71 @@ The {cert-manager-operator} is a cluster-wide service that provides application
 
 These release notes track the development of {cert-manager-operator}.
 
-:FeatureName: The {cert-manager-operator}
-include::snippets/technology-preview.adoc[leveloffset=+1]
-
 For more information, see xref:../../security/cert_manager_operator/index.adoc#cert-manager-operator-about[About the {cert-manager-operator}].
 
+[id="cert-manager-operator-release-notes-1.10.2"]
+== Release notes for {cert-manager-operator} 1.10.2
+
+// TODO: Confirm date closer to GA
+Issued: 2023-03-23
+
+The following advisory is available for the {cert-manager-operator} 1.10.2:
+
+* link:https://access.redhat.com/errata/RHEA-2023:1238[RHEA-2023:1238]
+
+For more information, see the link:https://cert-manager.io/docs/release-notes/release-notes-1.10[cert-manager project release notes for v1.10].
+
+[IMPORTANT]
+====
+If you used the Technology Preview version of the {cert-manager-operator}, you must uninstall and move all related resources for the Technology Preview version before installing this version of the {cert-manager-operator}.
+
+For more information, see xref:../../security/cert_manager_operator/cert-manager-operator-uninstall.adoc#cert-manager-operator-uninstall[Uninstalling the {cert-manager-operator}].
+====
+
+[id="cert-manager-operator-1.10.2-new-features-and-enhancements"]
+=== New features and enhancements
+
+This is the general availability (GA) release of the {cert-manager-operator}.
+
+* The following issuer types are supported:
+** ACME
+** CA
+** Self-signed
+
+* The following ACME challenge types are supported:
+** DNS-01
+** HTTP-01
+
+* The following DNS-01 providers for ACME issuers are supported:
+** Amazon Route 53
+** Azure DNS
+** Google Cloud DNS
+
+// TODO: Any other features to list or things to call out as supported? proxy support?
```

Comment:
@thejasn @swghosh I added a few things that we support to this list in the release notes. Is there anything else worthwhile that you think we should call out here?

Reply:
Both supporting cluster wide egress proxy and injecting custom CA are called out in bug fixes, so kind of becomes redundant.

Reply:
Both 
```
** Azure DNS
** Google Cloud DNS
```
Are supported using specific methods only, maybe we can add a footnote for this.

Reply:
@thejasn would it be better to call these features and move them up here? Or are they more accurate labeled as bug fixes?

As for your second comment, I do have the details listed in the known issues, but then also put that info in the main supported DNS providers section too. Let me know if that is sufficient: https://56754--docspreview.netlify.app/openshift-enterprise/latest/security/cert_manager_operator/cert-manager-operator-issuer-acme.html#cert-manager-acme-dns-providers_cert-manager-operator-issuer-acme